topdown/http_test.go

```diff
@@ -259,11 +266,25 @@ func TestHTTPRedirectEnable(t *testing.T) {
 		panic(err)
 	}
 
-	var testURL = "http://google.com"
 	data := loadSmallTestData()
 	rule := []string{fmt.Sprintf(
-		`p = x { http.send({"method": "get", "url": "%s", "enable_redirect": true}, x) }`, testURL)}
+		`p = x { http.send({"method": "get", "url": "%s", "enable_redirect": true}, x) }`, baseURL)}
 
 	// run the test
 	runTopDownTestCase(t, data, "http.send", rule, resultObj.String())
 }
+
+func getTestServer() (baseURL string, teardownFn func()) {
```

Comment:
[nit] There's no need to name the return arguments here, I think. (As long as we avoid _"naked `return`s"_, it doesn't matter much, though.